tests/app/views/handlers/test_view_submitted_response_pdf.py

```diff
@@ -0,0 +1,35 @@
+from datetime import datetime, timedelta, timezone
+from unittest.mock import Mock
+
+import pytest
+
+from app.data_models import QuestionnaireStore
+from app.questionnaire.questionnaire_schema import QuestionnaireSchema
+from app.views.handlers.view_submitted_response import ViewSubmittedResponseExpired
+from app.views.handlers.view_submitted_response_pdf import ViewSubmittedResponsePDF
+from tests.app.views.handlers.conftest import set_storage_data
```

Comment:
this could be `from .conftest import set_storage_data` ?

Reply:
coming from AWS lambda , it was a pain to have relative imports. very weird. If we use relative across let it be like that. TBH iam not fussed about it.